raiden/raiden_event_handler.py

```diff
@@ -374,12 +374,15 @@ def handle_contract_send_channelunlock(
             log.warning(
                 f'Unlock not done. '
                 f'Failed to find state/event that match current channel locksroots. '
+                f'chain_id:raiden.chain.network_id '
```

Comment:
@hackaugusto you forgot the `{ .. }` for the fstring here.

Reply:
AHH, thanks